README.md

```diff
@@ -40,7 +40,60 @@ For a detailed understanding of the system's architecture and design, refer to t
 
 ## Requirements and Setup
 
-*Insert a short description what is required to get your project running...*
+The application can be configured using environment variables. These variables are stored in a `.env` file at the root of the project.
+For configuring tests, there is a separate `.test.env` file.
+
+Here's a basic example of what the .env file could look like:
+
+```
+DB_USER=my_username
+DB_PASSWORD=my_password
+DB_ROOT_PASSWORD=my_password
+DB_NAME=heureka
+DB_ADDRESS=localhost
+DB_PORT=3306
+DB_SCHEMA=internal/database/mariadb/init/schema.sql
+
+DB_CONTAINER_IMAGE=mariadb:latest
+
+DOCKER_IMAGE_REGISTRY=hub.docker.com
+
+DOCKER_CREDENTIAL_STORE=docker-credential-desktop
```

Comment:
I think this is not a self-explanatory env var, lets add some context with a comment?